### ISSUE TITLE
Use proper names of guidelines

### ISSUE THREAD

**Issue by WilcoFiers** [act-rules/act-implementor#7]
On the rule implementation pages, the mapping key is used when the there is no title. Those work as a short hand, but are not particularly user friendly. Instead the full title of an accessibility requirement, and a link to that requirement should be used.
